opm/test_util/EclFilesComparator.cpp

```diff
@@ -108,15 +108,15 @@ void ECLFilesComparator::deviationsForCell(double val1, double val2, const std::
         std::cout << "(value1, value2) = (" << val1 << ", " << val2 << ")\n\n";
     }
     if (val1 < 0) {
-        if (abs(val1) > absTolerance) {
+        if (std::abs(val1) > absTolerance) {
```

Comment:
May I ask what problem the qualifier is supposed to solve?

In DUNE there have recently been [several commits](https://gitlab.dune-project.org/core/dune-grid/merge_requests/73) removing the std qualifiers. It is claimed that this has been done  to make [argument dependent lookup](https://en.wikipedia.org/wiki/Argument-dependent_name_lookup) work.